### ISSUE TITLE
To continue a finished training

### ISSUE THREAD

**Issue by Cons13411** [crazydonkey200/tensorflow-char-rnn#20]
hi. I am trying to train the char rnn on a dump of wikipedia. It has been divided in several files, and  I feed it one file at a time. The first file has been trained without any errors, but when it starts the second one by this command (python train.py --data_file=your-data-file --init_dir=your-output-folder) it returns lots of ( INFO:Unexpected char) error. I am not sure what is the problem. I trained the first file with this configuration
-data_file=./wiki2/enwiki-20180120-pages-articles-multistream.xml-0001.txt", "--hidden_size=512",
            "--embedding_size=300","--encoding=utf8", "--num_layers=3",
            "--model=rnn", "--num_epochs=10", "--dropout=0.5", "--test"
